packages/react-components/src/index.ts

```diff
@@ -55,7 +73,42 @@ export type {
   StrokeWidthTokens,
   Theme,
 } from '@fluentui/react-theme';
-export { SSRProvider } from '@fluentui/react-utilities';
+export {
+  clamp,
+  getNativeElementProps,
+  getNativeProps,
+  getPartitionedNativeProps,
+  getRTLSafeKey,
+  getSlots,
+  omit,
+  resolveShorthand,
+  shouldPreventDefaultOnKeyDown,
+  useBoolean,
+  useConst,
+  useControllableState,
+  useId,
+  useIsSSR,
+  useMergedEventCallbacks,
+  useMergedRefs,
+  useMount,
+  useSSRContext,
+  useTimeout,
+  useUnmount,
+  SSRContext,
```

Comment:
```suggestion
```

This is already used in `FluentProvider`  so I don't see much point in exporting this